bin/ipfs-api-mount

```diff
@@ -18,6 +18,8 @@ if __name__ == '__main__':
     parser.add_argument('--object-links-cache-size', type=int, default=256, help='Max number of object link sections kept in cache.')
     parser.add_argument('-b', '--background', action='store_true', help='Run in background.')
     parser.add_argument('-m', '--multithreaded', action='store_true', help='Use multiple threads to handle FS requests.')
+    parser.add_argument('--allow-other', action='store_true', help='Set fuse mount option \'allow_other\'')
+    parser.add_argument('--read-only', action='store_true', help='Set \'ro\' mount option instead of \'rw\'')
```

Comment:
I feel like `ro` should be always set, and theres no need to expose it as a parameter.